upgrade-operator/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "upgrade-operator"
+description = "Upgrade Operator"
+version = "0.1.0"
+edition = "2021"
+authors = ["Sahil Raja <sahil.raja@datacore.com>"]
```

Comment:
Nit - adding authors to a rust crate is no longer considered best practices in rust, and may eventually be deprecated and removed.